docs/cli.md

```diff
@@ -340,35 +340,51 @@ This won't show logs from running locally with `zapier test`, since those never
 
 ## migrate
 
-> Migrate users from one version of your integration to another.
+> Migrate a percentage of users or a single user from one version of your integration to another.
 
 **Usage**: `zapier migrate FROMVERSION TOVERSION [PERCENT]`
 
 Start a migration to move users between different versions of your integration. You may also "revert" by simply swapping the from/to verion strings in the command line arguments (i.e. `zapier migrate 1.0.1 1.0.0`).
 
-Only use this command to migrate users between non-breaking versions, use `zapier deprecate` if you have breaking changes!
+**Only use this command to migrate users between non-breaking versions, use `zapier deprecate` if you have breaking changes!**
 
 Migration time varies based on the number of affected Zaps. Be patient and check `zapier jobs` to track the status. Or use `zapier history` if you want to see older jobs.
 
 Since a migration is only for non-breaking changes, users are not emailed about the update/migration. It will be a transparent process for them.
 
-We recommend migrating a small subset of users first, then watching error logs of the new version for any sort of odd behavior. When you feel confident there are no bugs, go ahead and migrate everyone. If you see unexpected errors, you can revert.
+We recommend migrating a small subset of users first, via the percent argument, then watching error logs of the new version for any sort of odd behavior. When you feel confident there are no bugs, go ahead and migrate everyone. If you see unexpected errors, you can revert.
 
-You can migrate a single user by using `--user` (i.e. `zapier migrate 1.0.0 1.0.1 --user=user@example.com`).
+You can migrate a specific user's Zaps by using `--user` (i.e. `zapier migrate 1.0.0 1.0.1 --user=user@example.com`). This will migrate Zaps in any account the user is part of where the following criteria is met.  
+
+  - The Zap is owned by the user.  
+
+  - The Zap is not shared.  
+
+  - The integration auth used is not shared.  
+
+Alternatively, you can pass the `--account` flag, (i.e. `zapier migrate 1.0.0 1.0.1 --account=account@example.com`). This will migrate all the user's Zap steps in any account the user is part of, regardless of Zap owner.
```

Comment:
It's probably clearer and more accurate saying something like `This will migrate all users' Zaps within all accounts for which the specified user is a member`